app/views/layouts/_top_bar.html.erb

```diff
@@ -13,7 +13,16 @@
     <%= link_to I18n.t('spree.login'), login_path, class: 'auth-link' %>
   <% end %>
 
+<!--  https://trello.com/c/CQKgoAUG/13-add-a-link-to-the-cart-to-the-header-->
   <div class="cart-link" id="link-to-cart" data-top-bar-target="cartLink">
+    <%= link_to cart_path do %>
+      <div class="cart-info" id="cart-info"> <!-- заменили класс badg на cart-info -->
```

Comment:
Please, do not use Russian comments.

Reply:
I apologize. It was for myself, as there were a lot of changes. To make it easier to rollback. I forgot to remove this comment.